src/main/java/net/minecraftforge/fluids/FluidAttributes.java

```diff
@@ -355,8 +496,17 @@ public static Builder builder(ResourceLocation stillTexture, ResourceLocation fl
         private int density = 1000;
         private int temperature = 300;
         private int viscosity = 1000;
-        private boolean isGaseous;
         private Rarity rarity = Rarity.COMMON;
+        private ToDoubleBiFunction<FluidState, Entity> motionScale = (state, entity) -> 0.014D;
+        private ToDoubleBiFunction<FluidState, Entity> fallDistanceModifier = (state, entity) -> 0.0D;
+        private Predicate<FluidState> canSwim = state -> state.is(FluidTags.WATER);
+        private BiPredicate<FluidState, LivingEntity> canDrown = (state, entity) -> state.is(FluidTags.WATER);
+        private BiPredicate<FluidState, Entity> canExtinguish = (state, entity) -> state.is(FluidTags.WATER);
+        private BiPredicate<FluidState, BlockState> canHydrate = (fluidState, blockState) -> fluidState.is(FluidTags.WATER);
+        private BiFunction<FluidState, LivingEntity, Float> horizontalMotionModifier = (state, entity) -> entity.isSprinting() ? 0.9F : 0.8F;
+        private BiFunction<Enchantment, LivingEntity, Float> enchantmentModifier = (enchantment, entity) -> enchantment instanceof WaterWalkerEnchantment ? EnchantmentHelper.getDepthStrider(entity) : 0F;
```

Comment:
I am not quite sure if this is the correct default for the enchantmentModifier or if this should be explicitly just set on forge's water fluid attributes as the default, as from what I can see of the default behavior vanilla has:
isInWater -> getDepthStrider, but this default would make someone seeing what enchantment modifiers lava has for example return it supports it instead of not. Alternatively if the overall consensus is that custom fluids should default to supporting depth strider, then potentially lava's fluid attribute should explicitly just return zero as the enchantment modifier for this.